public/css/widget/state-change.less

```diff
@@ -32,3 +32,12 @@
     }
   }
 }
+
+.state-change.detail-history {
+  .state-ball.ball-size-m {
+    height: 1em;
+    width: 1em;
+    margin-top: .26em;
+    margin-left: -.375em;
```

Comment:
Why this? `ball-size-m` already has a `margin-left: -.275em`. Isn't that enough? Disabling this in the dev tools changes nothing for me.

Reply:
You need to take a closer look at the state balls for soft state change, then it should be clear why it is needed.

![Bildschirmfoto 2021-10-01 um 10 11 12](https://user-images.githubusercontent.com/57616252/135587751-6eceb646-1a0d-4fa1-8426-4db8965e57d8.png) ![Bildschirmfoto 2021-10-01 um 10 13 25](https://user-images.githubusercontent.com/57616252/135587771-923ed317-647f-4236-b7fa-5cc179d34707.png)